src/js/components/deployments/deployment-report/deploymentdevicelistitem.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import Time from 'react-time';
 
 // material ui
 import { Button, LinearProgress, TableCell, TableRow } from '@material-ui/core';
 
 import { formatTime } from '../../../helpers';
 import DeviceIdentityDisplay from '../../common/deviceidentity';
+import { LocaleTime } from '../../common/localetime';
```

Comment:
Some of these are "named imports", some import the default (aka. no curly braces) - please pick one for consistency. As to which: if there is just a single component/ function/ object/ ... in a module, I usually stick to importing the default; if there are multiple things of interest "it depends"... but in general, I think consistency helps with navigation 👌